test/unittests/commands/xmlupload/make_rdf_graph/test_make_file_value.py

```diff
@@ -0,0 +1,204 @@
+import pytest
+import regex
+from rdflib import RDF
+from rdflib import XSD
+from rdflib import BNode
+from rdflib import Literal
+
+from dsp_tools.commands.xmlupload.make_rdf_graph.constants import ARCHIVE_FILE_VALUE
+from dsp_tools.commands.xmlupload.make_rdf_graph.constants import AUDIO_FILE_VALUE
+from dsp_tools.commands.xmlupload.make_rdf_graph.constants import DOCUMENT_FILE_VALUE
+from dsp_tools.commands.xmlupload.make_rdf_graph.constants import KNORA_API
+from dsp_tools.commands.xmlupload.make_rdf_graph.constants import MOVING_IMAGE_FILE_VALUE
+from dsp_tools.commands.xmlupload.make_rdf_graph.constants import STILL_IMAGE_FILE_VALUE
+from dsp_tools.commands.xmlupload.make_rdf_graph.constants import TEXT_FILE_VALUE
+from dsp_tools.commands.xmlupload.make_rdf_graph.make_file_value import _add_metadata
+from dsp_tools.commands.xmlupload.make_rdf_graph.make_file_value import _get_file_type_info
+from dsp_tools.commands.xmlupload.make_rdf_graph.make_file_value import _make_abstract_file_value_graph
+from dsp_tools.commands.xmlupload.make_rdf_graph.make_file_value import make_file_value_graph
+from dsp_tools.commands.xmlupload.make_rdf_graph.make_file_value import make_iiif_uri_value_graph
+from dsp_tools.commands.xmlupload.models.deserialise.xmlresource import BitstreamInfo
+from dsp_tools.commands.xmlupload.models.permission import Permissions
+from dsp_tools.commands.xmlupload.models.permission import PermissionValue
+from dsp_tools.commands.xmlupload.models.rdf_models import AbstractFileValue
+from dsp_tools.commands.xmlupload.models.rdf_models import FileValueMetadata
+from dsp_tools.commands.xmlupload.models.rdf_models import RDFPropTypeInfo
+from dsp_tools.models.exceptions import BaseError
+
+
+@pytest.fixture
+def metadata_permissions() -> FileValueMetadata:
+    return FileValueMetadata("permissions")
+
+
+@pytest.fixture
+def metadata_no_permissions() -> FileValueMetadata:
+    return FileValueMetadata(None)
+
+
+@pytest.fixture
+def abstract_file_permissions(metadata_permissions: FileValueMetadata) -> AbstractFileValue:
+    return AbstractFileValue("value", metadata_permissions)
+
+
+@pytest.fixture
+def abstract_file_no_permissions(metadata_no_permissions: FileValueMetadata) -> AbstractFileValue:
+    return AbstractFileValue("value", metadata_no_permissions)
+
+
+class TestIIIFURI:
+    def test_make_iiif_uri_value_graph_with_permissions(self, abstract_file_permissions: AbstractFileValue) -> None:
+        res_bn = BNode()
+        g, _ = make_iiif_uri_value_graph(abstract_file_permissions, res_bn)
```

Comment:
Don't you think this is necessary to assert?

```suggestion
        g, file_prop = make_iiif_uri_value_graph(abstract_file_permissions, res_bn)
        assert file_prop == KNORA_API.hasStillImageFileValue
```

Reply:
No, because I will remove it once we can do real json-ld, this was just for the frame.
If the correct property is not used line 54 would fail